models/lpjguess/NAMESPACE

```diff
@@ -1,3 +1,4 @@
 # Generated by roxygen2 (4.0.2): do not edit by hand
 
+export(model2netcdf.LPJGUESS)
```

Comment:
@araiho if you don't export a function it's not available to users when you load the package, but rather can only be used internally by other functions in the package